webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const devCerts = require("office-addin-dev-certs");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+module.exports = async (env, options) => {
+    const dev = options.mode === "development";
+    const config = {
+        devtool: "source-map",
+        entry: {
+            commands: "./src/commands/commands.ts",
+            fallbackauthtaskpane: "./src/taskpane/fallbackauthtaskpane.ts",
+            polyfill: "@babel/polyfill",
+            taskpane: "./src/taskpane/taskpane.ts",
+        },
+        resolve: {
+            extensions: [".ts", ".tsx", ".html", ".js"]
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.ts$/,
+                    exclude: /node_modules/,
+                    use: "babel-loader"
+                },
+                {
+                    test: /\.tsx?$/,
```

Comment:
If I understand this line (far from a sure thing), ts-loader will apply to both .tsx and .ts file. The rule just above applies babel-loader to .ts files. So both loaders will apply to .ts files; first ts-loader and then it's output is passed to babel-loader. Am I understanding this correctly?

Reply:
I believe so. I will let @TCourtneyOwen confirm